ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -907,6 +911,43 @@ export default class SecurityTab extends PureComponent {
     );
   }
 
+  ///: BEGIN:ONLY_INCLUDE_IF(transaction-simulation)
+  renderSimulationsToggle() {
+    const { t } = this.context;
+    const { useSimulations, setUseSimulations } = this.props;
+
+    return (
+      <Box
```

Comment:
Each time we add one of these, I wish all this duplication was in a `SecurityToggleButton` function to massively shrink this file.

But as usual, not worth the time or risk at this point 😄

Reply:
Exactly Matthew, I was like amazed all those repetition
It's been fixed in the onboarding privacy settings, might touch this later :)